### PR TITLE
Repositioning Tooltip for responsive sites on window resize

### PR DESCRIPTION
**Pull request by ipiz**
For responsives site, I moved onresize function on _showElement and call on resize + _placeTooltip for repositioning on responsive sites

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "intro.js",
     "description": "Better introductions for websites and features with a step-by-step guide for your projects",
-    "version": "2.0",
+    "version": "2.0.0",
```
Comment (afshinm) [inline package.json]: Can you please remove this change?

File: package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         "node-minify": "*"
     },
     "engine": [
-        "node >=0.1.90"
+        "node >=2.10.1"
```
Comment (afshinm) [inline package.json]: and also this one.

Comment (ipiz) [inline package.json]: Done, thanks.

**Comment by Kaain** [comment]
I tried to use this fix for Version 2.3.0 but unfortunately it's only working for 2.0.0. Is there any chance that this issue is fixed in the next version? Or has anyone a working fix for 2.3.0?

**Comment by afshinm** [comment]
nice one but doesn't work properly with latest version. could you please have a look?

**Comment by afshinm** [comment]
Fixed.

**Pull request closed without merging**